### PR TITLE
Enter keymap

### PR DESCRIPTION
**Pull request by theHamsta**
I always want to press `<enter>` to go to the current node in AST in the code buffer. Don't know whether this keymap can be added...

### CODE REVIEW

**Review by vigoux — APPROVED** [review]

**Comment by theHamsta** [comment]
Maybe I should the case when the code buffer is not in any window and bring it to front...

File: lua/nvim-treesitter-playground/internal.lua [hunk 1]
```diff
@@ -264,6 +265,13 @@ function M.highlight_nodes(bufnr, nodes)
   end
 end
 
+function M.goto_node(bufnr)
+  utils.for_each_buf_window(bufnr, function(window)
+    api.nvim_set_current_win(window)
+    return
```
Comment (steelsojka) [inline lua/nvim-treesitter-playground/internal.lua:271]: /nit Is the return needed here?

**Review by steelsojka — APPROVED** [review]

File: lua/nvim-treesitter-playground/internal.lua (hunk 1 above)
Comment (theHamsta) [reply lua/nvim-treesitter-playground/internal.lua:271]: not really. if you remove it cursor would go to last window of the buffer instead of the first.

Comment (steelsojka) [reply lua/nvim-treesitter-playground/internal.lua:271]: But the return is for the callback though. The return value isn't used in `for_each_buf_window`.

Comment (theHamsta) [reply lua/nvim-treesitter-playground/internal.lua:271]: :facepalm: right!

File: lua/nvim-treesitter-playground/internal.lua [hunk 2]
```diff
@@ -264,6 +269,23 @@ function M.highlight_nodes(bufnr, nodes)
...
+  local bufwin = vim.fn.win_findbuf(bufnr)[1]
+  if bufwin then
+    api.nvim_set_current_win(bufwin)
+  else
+    local node = M.get_current_node(bufnr)
+
+    local win = api.nvim_get_current_win()
+    vim.cmd "vsplit"
+    api.nvim_win_set_buf(win, bufnr)
+    api.nvim_set_current_win(win)
```
Comment (theHamsta) [inline lua/nvim-treesitter-playground/internal.lua:282]: This is written is a bit convoluted way to get the split in the other direction as the playground would open.

**Review by steelsojka — APPROVED** [review]
Looks good to me!

**Pull request merged**